src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetWidget.java

```diff
@@ -351,7 +351,7 @@ private Toolbar createToolbar(TextFileType fileType)
       toolbar.addLeftWidget(previewHTMLButton_ =
          mgr.getSourceCommand(commands_.previewHTML(), column_).createToolbarButton());
       knitDocumentButton_ =
-         mgr.getSourceCommand(commands_.knitDocument(), column_).createToolbarButton();
+         mgr.getSourceCommand(commands_.knitDocument(), column_).createToolbarButton(false);
```

Comment:
I'm not a big fan of boolean arguments like these (though I know we do it a lot, and I've written a bunch of code that does it, too). So this is absolutely just me thinking out loud, no real need to change anything.

Alternatives include creating a separate method, e.g.

`.createUnsyncedToolbarButton()`

Commenting the argument:

`.createToolbarButton(false/*synced*/)`

Pass an enum, instead (`public enum SyncType { UNSYNCED, SYNCED }`).

Reply:
I like `.createUnsyncedToolbarButton` and switched to that.